docs/api.md

```diff
@@ -794,6 +795,45 @@ Enables Dark Mode support on macOS 10.14+.
 
 See https://electronjs.org/docs/api/browser-window#setting-backgroundcolor
 
+#### [bookmarks-menu]
+
+```
+--bookmarks-menu <string>
+```
+
+Path to a JSON file containing configuration for a predefined bookmarks menu. In addition to containing a list of bookmarks, this file customizes the name of the menu and (optionally) allows assigning keyboard shortcuts to bookmarks.
+
+Example of such a JSON file:
+
+```json
+{
+    "menuLabel": "Music",
+    "bookmarks": [
+        {
+            "title": "lofi.cafe",
+            "url": "https://lofi.cafe/",
+            "type": "link",
+            "shortcut": "Cmd+1"
```

Comment:
Please summarize (before the example) the supported modifier keys, and link to electron docs

Reply:
Done in 27f6af6